food-waste-ui/src/components/MarketDetail/MarketDetail.jsx

```diff
@@ -51,8 +53,37 @@ export default function MarketDetail({
                   {curProduct[0].description}
                 </div>
                 <div className="product-quantity">{curProduct[0].quantity}</div>
+                <div className="product-dates">
+                  <p>Post Created on: July 17, 2022</p>
```

Comment:
Is this test information? If so, remove. Each commit should not have any testing related code (unless its a unit/e2e test!)